data/helpers.d/backend

```diff
@@ -51,3 +51,41 @@ ynh_remove_logrotate () {
 		sudo rm "/etc/logrotate.d/$app"
 	fi
 }
+
+# Create a dedicated nginx config
+#
+# usage: ynh_add_nginx_config
+ynh_add_nginx_config () {
+	finalnginxconf="/etc/nginx/conf.d/$domain.d/$app.conf"
+	ynh_compare_checksum_config "$finalnginxconf" 1
```

Comment:
So with the new naming `ynh_backup_if_checksum_is_different`, it is clearer: we backup the file if it was modified, and then proceed to overwrite it, don't we?